### PR TITLE
Document tryNext Gradle property

### PR DESCRIPTION
**Pull request by sarahhaggarty**
This PR documents the tryNext Gradle property as part of Kotlin 2.0.0.

### CODE REVIEW

File: docs/topics/gradle/gradle-compilation-and-caches.md [hunk 1]
```diff
@@ -417,6 +418,13 @@ tasks.named("compileKotlin").configure {
 
 If there is insufficient memory to run the compilation, you can see a message about it in the logs.
 
+## Trying the latest language version
+
+Starting with Kotlin 2.0.0, to try the latest language version, set the `kotlin.experimental.tryNext` property in your `gradle.properties`
+file. When you use this property, the Kotlin Gradle plugin increments the language version to one above the default value
```
[the remark this answers is not in the document]
Comment (sarahhaggarty) [reply docs/topics/gradle/gradle-compilation-and-caches.md:418]: Do you mean that the language version used is included in the output of build reports per task?

File: docs/topics/gradle/gradle-compilation-and-caches.md [hunk 2]
```diff
@@ -417,6 +418,13 @@ tasks.named("compileKotlin").configure {
 
 If there is insufficient memory to run the compilation, you can see a message about it in the logs.
 
+## Trying the latest language version
+
+Starting with Kotlin 2.0.0, to try the latest language version, set the `kotlin.experimental.tryNext` property in your `gradle.properties`
```
[the remark this answers is not in the document]
Comment (sarahhaggarty) [reply docs/topics/gradle/gradle-compilation-and-caches.md:417]: Added!

File: docs/topics/gradle/gradle-compilation-and-caches.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Tapchicoma) [reply docs/topics/gradle/gradle-compilation-and-caches.md:418]: yes, report is looking something like this:
```
##### 'kotlin.experimental.tryNext' results #####
:app:compileKotlin: $nextKotlinLanguageVersion language version
:lib:compileKotlin: $nextKotlinLanguageVersion language version
##### 100% (2/2) tasks have been compiled with Kotlin $nextKotlinLanguageVersion #####
```

[the remark this answers is not in the document]
Comment (sarahhaggarty) [reply docs/topics/gradle/gradle-compilation-and-caches.md:418]: Added a sentence about build reports :)

**Review by Tapchicoma — APPROVED** [review]

**Pull request merged**